modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/power_of_attorney_requests_controller.rb

```diff
@@ -3,6 +3,13 @@
 module AccreditedRepresentativePortal
   module V0
     class PowerOfAttorneyRequestsController < ApplicationController
+      module Statuses
+        ALL = [
+          PENDING = 'pending',
+          COMPLETED = 'completed'
+        ].freeze
+      end
```

Comment:
Please remove the Statuses module and use the constants directly

```suggestion
 PENDING = 'pending'.freeze
 COMPLETED = 'completed'.freeze
 ALL = [PENDING, COMPLETED].freeze
```

Reply:
This branch no longer contains that code - base has been changed to master